tests/x11/reboot_gnome.pm

```diff
@@ -40,12 +40,19 @@ sub run() {
 
         send_key "ret";
 
-        wait_still_screen 8, 8;    # wait 8 seconds to make authentication window disappear after successful authentication
```

Comment:
we verified yesterday: the window turns black after 10 seconds on zkvm. So having the check_var for BACKEND is good enough - as long as we don't have the same need for workaround on zkvm.